lib/active_job/queue_adapters/pubsub_adapter.rb

```diff
@@ -4,18 +4,29 @@ module ActiveJob
   module QueueAdapters
     class PubsubAdapter
       # Enqueue a job to be performed.
-      #
       # @param [ActiveJob::Base] job The job to be performed.
       def enqueue(job)
-        raise(NotImplementedError)
+        # Rails.logger.info("[PubsubAdapter] Enqueue: #{job.arguments[0]}")
+        Base.execute(job.serialize)
```

Comment:
This would defeat the purpose of background processing. You need to publish to GCP here.